visualisation/package.json

```diff
@@ -2,40 +2,40 @@
   "name": "digital-twin",
   "version": "0.0.1",
   "private": true,
+  "type": "module",
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "lint": "eslint src --ext .js,.ts,.vue"
+    "serve": "vite",
+    "build": "run-p type-check \"build-only {@}\" --",
+    "preview": "vite preview",
+    "build-only": "vite build",
+    "type-check": "vue-tsc --build --force",
+    "lint": "eslint src --ext .js,.ts,.vue --fix --ignore-path .gitignore",
```

Comment:
Ensure warnings are shown as errors in linting, ensure fixes are pushed by actions, and ensure type-check is performed by action